src/main/java/com/training/library/service/DatabaseService.java

```diff
@@ -0,0 +1,95 @@
+package com.training.library.service;
+
+import com.training.library.entity.Book;
+import java.lang.reflect.InvocationTargetException;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class DatabaseService {
+
+    private final String url = "jdbc:mysql://localhost:3306/library?serverTimezone=Europe/Moscow&useSSL=false";
+    private final String username = "root";
+    private final String password = "12345";
+
+    public DatabaseService() {
+    }
+
+    public Connection setConnection() throws
+            ClassNotFoundException,
+            NoSuchMethodException,
+            IllegalAccessException,
+            InvocationTargetException,
+            InstantiationException,
+            SQLException {
+        Class.forName("com.mysql.cj.jdbc.Driver").getDeclaredConstructor().newInstance();
```

Comment:
Consider to change reflection call to `Driver mySqlDriver = new Driver(); DriverManager.registerDriver(mySqlDriver);` and this logic can be moved to constructor as we need register driver only 1 time. It would be nice to make DatabaseService object is common for all dependencies.